srcpkgs/wezterm/patches/7b904f05eb4de9ed44d2068355a571b117eba0e1.patch

```diff
@@ -0,0 +1,232 @@
+From 7b904f05eb4de9ed44d2068355a571b117eba0e1 Mon Sep 17 00:00:00 2001
```

Comment:
I would call this patch something like 32bit.patch, the commit has doesn't tell us anything useful.

Reply:
Ok, I've renamed both to be more similar to git exports; as well as ensure a reproduced ordering of patches.